Package@swift-5.8.swift

```diff
@@ -0,0 +1,56 @@
+// swift-tools-version:5.8
```

Comment:
can this be 5.6 again?

Reply:
Just added a separate `Package@swift-5.7.swift` to keep Xcode 14.1 compatibility (the min to submit to the App Store), and added a CI job to verify that.